indico/modules/events/registration/clone_test.py

```diff
@@ -0,0 +1,43 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2020 CERN
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see the
+# LICENSE file for more details.
+
+from indico.core.db import db
+from indico.modules.events.cloning import EventCloner
+from indico.modules.events.features.util import set_feature_enabled
+from indico.modules.events.registration.models.registrations import Registration
+
+
+pytest_plugins = 'indico.modules.events.registration.testing.fixtures'
+
+
+def test_registration_clone(dummy_event, dummy_regform, create_event, dummy_user):
+    set_feature_enabled(dummy_event, 'registration', True)
+
+    reg1 = Registration(
+        event=dummy_event,
+        event_id=dummy_event.id,
+        registration_form_id=dummy_regform.id,
+        first_name="Guinea",
+        last_name="Pig",
+        checked_in=True,
+        state=1,
+        currency="USD",
+        email="1337@example.com",
+        user=dummy_user
+    )
+    db.session.flush()
+
+    assert dummy_user.registrations.filter_by(event=dummy_event).first().full_name == 'Guinea Pig'
+    assert dummy_event.registration_forms[0].active_registrations[0].user == dummy_user
+    assert dummy_event.registration_forms[0].active_registrations[0].checked_in is True
+
+    copied_event = create_event()
+    EventCloner.run_cloners(dummy_event, copied_event, {'registrations', 'registration_forms'})
+
+    assert dummy_user.registrations.filter_by(event=copied_event).first().full_name == 'Guinea Pig'
+    assert copied_event.registration_forms[0].active_registrations[0].user == dummy_user
+    assert copied_event.registration_forms[0].active_registrations[0].checked_in is False
```

Comment:
`assert not ...`

Reply:
I'd assign `copied_event.registration_forms[0].active_registrations[0]` to `copied_reg` or something like that